core/src/main/java/jenkins/security/ImpersonatingUserDetailsService.java

```diff
@@ -18,19 +24,47 @@
  * @author Kohsuke Kawaguchi
  */
 public class ImpersonatingUserDetailsService implements UserDetailsService {
+    
+    private static final Logger LOGGER = Logger.getLogger(ImpersonatingUserDetailsService.class.getName());
+    
     private final UserDetailsService base;
 
+    @Restricted(NoExternalUse.class)
+    public static /* Script Console modifiable */ boolean DISABLE_CACHE_FOR_IMPERSONATION = 
```

Comment:
unused outside of this class.

```suggestion
    private static boolean DISABLE_CACHE_FOR_IMPERSONATION = 
```